package.json

```diff
@@ -85,12 +85,15 @@
     "prism-react-renderer": "1.3.5",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "ts-node": "10.8.2",
-    "typedoc": "0.23.5",
+    "ts-node": "10.9.1",
+    "typedoc": "0.23.10",
     "typescript": "4.7.4",
     "url-loader": "4.1.1"
   },
   "publishConfig": {
     "tag": "alpha"
+  },
+  "dependencies": {
+    "npm-check": "^6.0.1"
   }
```

Comment:
```suggestion
  }
```

Reply:
@saihaj Thanks, good catch from you and @yaacovCR
Just forgot to add `--global`, fixed.